data/brands/amenity/fast_food.json

```
@@ -4995,6 +4995,19 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Robeks",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "fast_food",
```

```suggestion
        "alt_name": "Robeks Juice",
        "amenity": "fast_food",
```
Based on their Wikipedia article it looks like some of their stores are signed as 'Robeks Juice'